assets/js/buttons-examples.js

```diff
@@ -199,6 +199,12 @@ $(document).ready(function () {
           classes: "scroll-to-top-btn",
           text: "",
         },
+        {
+          type: "close",
+          label: "Close Button",
+          classes: "close-btn",
+          text: "",
+        }
```

Comment:
Please change this object to the following:

```
        {
          type: "close",
          label: "Close Button",
          classes: "",
          variations: [
             {
                 classes: "close-btn",
                 label: ""
             }
          ]
        }
```

This will make the close button appear without the button colors, just on its own.

Reply:
Also place this button under Icon in the correct alphabetical order